extensions/amazon-lambda-http/deployment/src/main/resources/http/sam.jvm.yaml

```diff
@@ -17,6 +17,10 @@
         MemorySize: 512
         Policies: AWSLambdaBasicExecutionRole
         Timeout: 15
+        Environment:
+          Variables:
+            QUARKUS_AWS_LAMBDA_FORCE_USER: false
```

Comment:
This should be in the native template too.

Reply:
You can test a native binary with SAM LOCAL as well, correct?  So why wouldn't it be there too?